client/settings/survey-modal/style.scss

```diff
@@ -0,0 +1,35 @@
+.survey-section {
+	.components-modal__header {
+		margin-bottom: 16px;
```

Comment:
Should we remove this? Otherwise the spacing will be inconsistent with the other components.

I understand that Figma has some different spacing on this, but all the other components don't. It looks like sometimes the spacing in Figma is referring to a different version of Gutenberg, causing inconsistent spacing.
We should not override the spacing from the Gutenberg components, otherwise they might need to be adjusted on an upgrade of Gutenberg.

Reply:
Good point. The bottom margin seemed just _slightly_ too much for my eyes which left too much space for the text but leaving it alone doesn't make much of a difference.